src/overlays/actors/ovl_Bg_Ingate/z_bg_ingate.c

```diff
@@ -5,6 +5,8 @@
  */
 
 #include "z_bg_ingate.h"
+#include <ultra64.h>
+#include <global.h>
```

Comment:
these are already included in the z_bg_ingate.h header so these are uneeded

Reply:
👍